plugins/org.yakindu.sct.model.sexec/src/org/yakindu/sct/model/sexec/naming/tree/TreeNamingService.xtend

```diff
@@ -345,35 +348,27 @@ class TreeNamingService implements INamingService {
 	def protected List<String> suffix(Vertex it) {
 		return new ArrayList<String>();
 	}
-	
+
 	def protected List<String> addSeparator(List<String> segments) {
 		val List<String> result = newArrayList
-		for(var i = 0; i < segments.size(); i++) {
-			result.add(segments.get(i)) {
-				if(i < segments.size() - 1) {
+		for (var i = 0; i < segments.size(); i++) {
+			result.add(segments.get(i))
+			{
```

Comment:
This is not our usual formatting style

Reply:
Or in other words, you can just remove the brackets

Reply:
Yeah that's old code, I didn't reformat it all.